Assets/VRTK/SDK/Daydream/SDK_DaydreamController.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Daydream Controller|SDK_Daydream|003
+namespace VRTK
+{
+#if VRTK_SDK_DAYDREAM
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    public class SDK_DaydreamController : SDK_FallbackController
+    {
+        private GameObject controller;
+
+        private Vector3 prevPosition;
+        private Vector3 prevRotation;
+        private Vector3 velocity = Vector3.zero;
+        private Vector3 angularVelocity = Vector3.zero;
+
+        //TODO: where should we check that controller is connected? eg if (GvrController.State != GvrConnectionState.Connected) 
+
+        public override void ProcessUpdate(uint index, Dictionary<string, object> options)
+        {
+            if (controller == null)
+            {
+                return;
+            }
+
+            velocity = (controller.transform.position - prevPosition) / Time.deltaTime;
+
+            angularVelocity = Quaternion.FromToRotation(prevRotation, controller.transform.eulerAngles).eulerAngles / Time.deltaTime;
+
+            prevPosition = controller.transform.position;
+            prevRotation = controller.transform.rotation.eulerAngles;
+        }
+
+        public override string GetControllerDefaultColliderPath(ControllerHand hand)
+        {
+            var returnCollider = "ControllerColliders/Fallback";
+            return returnCollider;
+        }
+
+        public override string GetControllerElementPath(ControllerElements element, ControllerHand hand, bool fullPath = false)
+        {
+            //TODO: Use Gvr's tooltips or add an attach object ourselves
+            string dd = "Controller/ddcontroller/";
+            string pad = dd + "Tooltips/TouchPadOutside";
+            string app = dd + "Tooltips/AppButtonOutside";
+
+            switch (element)
+            {
+                case ControllerElements.AttachPoint:
+                    return pad; //TODO: attach point at tip of controller?
+                case ControllerElements.Touchpad:
+                    return pad;
+                case ControllerElements.ButtonOne:
+                    return app;
+                default:
+                    return dd;
+            }
+        }
+
+
+        public override uint GetControllerIndex(GameObject controller)
+        {
+            uint index = 0;
+
+            switch (controller.name)
+            {
+                case "Camera":
+                    index = 0;
+                    break;
+                case "RightController":
+                    index = 1;
+                    break;
+                case "LeftController": //Will be ignored; REMOVE OR RETURN 1 ?
```

Comment:
return `uint.maxValue` as that is the common usage in the other SDKs for a controller that cannot be found.